js/main.js

```diff
@@ -0,0 +1,24 @@
+// Доработано на основе: https://dev-gang.ru/article/javascript-generirovat-sluczainoe-czislo-v-diapazone-s1azn94f4k/
+const GET_RANDOM_NUMBER = (min, max) => {
+  if (min > max) {
+    const SWAP = min;
+    min = max;
+    max = SWAP;
+  }
+  const RANDOM_NUMBER = Math.random()*((max+1)-min) + min;
```

Comment:
Между операндами всегда пробелы не забываем
```suggestion
  const RANDOM_NUMBER = Math.random() * ((max + 1) - min) + min;
```